spec.html

```diff
@@ -35,10 +35,10 @@ <h1>Syntax</h1>
       PipelineExpression[In, Yield, Await] :
         LogicalORExpression[?In, ?Yield, ?Await]
         [~Await] PipelineExpression[?In, ?Yield, ?Await] `|>` LogicalORExpression[?In, ?Yield, ?Await]
-        [+Await] PipelineExpression[?In, ?Yield, ?Await] `|>` [lookahead &lt;! {`await`}] LogicalORExpression[?In, ?Yield, ?Await]
-        [+Await] PipelineExpression[?In, ?Yield, ?Await] `|>` `await` LogicalORExpression[?In, ?Yield, ?Await]
```

Comment:
Deleting the second line looks good, but deleting the first line makes pipeline not work in async functions at all.

Reply:
I'll re-add it, but could you elaborate as to why that's the case?